### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

File: com.anydesk.Anydesk.appdata.xml [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
+  <summary xml:lang="es">Conéctese a un ordenador de forma remota</summary>
   <metadata_license>CC0-1.0</metadata_license>
   <url type="homepage">https://www.anydesk.com</url>
-  <description>
-    <p>Working together? Accessing your apps from the road? Showing family videos to your friends?
-    No problem with AnyDesk. And your data stays at home.</p>
-    <p>AnyDesk is the fastest remote desktop software on the market. It allows for new usage scenarios and applications that have not been possible with current remote desktop software.</p>
-  </description>
+  <description><p>AnyDesk ensures secure and reliable remote desktop connections for IT professionals and on-the-go individuals alike.</description>
+  <description xml:lang="pt_BR">A AnyDesk assegura ligações de computador remoto seguras e fiáveis tanto para profissionais de TI como para pessoas em viagem.<p></p></description>
```
Comment (nedrichards) [inline com.anydesk.Anydesk.appdata.xml]: Paragraph tags are not correct here.

**Comment by nedrichards** [comment]
bot, build

File: com.anydesk.Anydesk.appdata.xml [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+  <summary xml:lang="pt_BR">Conectar com um computador remotamente</summary>
+  <summary xml:lang="es">Conéctese a un ordenador de forma remota</summary>
   <metadata_license>CC0-1.0</metadata_license>
   <url type="homepage">https://www.anydesk.com</url>
-  <description>
-    <p>Working together? Accessing your apps from the road? Showing family videos to your friends?
-    No problem with AnyDesk. And your data stays at home.</p>
-    <p>AnyDesk is the fastest remote desktop software on the market. It allows for new usage scenarios and applications that have not been possible with current remote desktop software.</p>
-  </description>
+  <description><p>AnyDesk ensures secure and reliable remote desktop connections for IT professionals and on-the-go individuals alike.</p></description>
```
Comment (nedrichards) [inline com.anydesk.Anydesk.appdata.xml]: I think we should put the following language into a new paragraph here as well (similar to the other extra-data applications.)

`<p>NOTE: This wrapper is not verified by, affiliated with, or supported by AnyDesk.</p>`

File: com.anydesk.Anydesk.appdata.xml [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
-  </description>
+  <description><p>AnyDesk ensures secure and reliable remote desktop connections for IT professionals and on-the-go individuals alike.</p><p>NOTE: This wrapper is not verified by, affiliated with, or supported by Adobe.</p></description>
+  <description xml:lang="pt_BR"><p>A AnyDesk assegura ligações de computador remoto seguras e fiáveis tanto para profissionais de TI como para pessoas em viagem.</p><p>NOTA: Este aplicativo não foi empacotado pelo AnyDesk</p></description>
+  <description xml:lang="es"><p>Gracias a AnyDesk, contará con conexiones seguras y fiables de escritorio remoto para profesionales informáticos y usuarios en movimiento.</p><p>NOTA: Esta aplicación no ha sido empaquetado por AnyDesk</p></description>
   <screenshots>
-    <screenshot>
-      <image type="source">https://anydesk.com/media/images/image-sshot-linux-main-en_scXhgcK.width-990.png</image>
-    </screenshot>
+    <screenshot><image type="source">https://anydesk.pt/media/images/image-sshot-windows-main-en.width-990.png</image></screenshot>
```
Comment (nedrichards) [inline com.anydesk.Anydesk.appdata.xml]: We want to use Linux screenshots rather than windows (see the one you're replacing as an example)

Comment (LeandroStanger) [reply com.anydesk.Anydesk.appdata.xml]: No images from the Linux version
Windows only, Mec OS, IOS and Android

File: com.anydesk.Anydesk.appdata.xml [hunk 4]
```diff
@@ -1,21 +1,19 @@
...
+  <summary xml:lang="pt_BR">Conectar com um computador remotamente</summary>
+  <summary xml:lang="es">Conéctese a un ordenador de forma remota</summary>
   <metadata_license>CC0-1.0</metadata_license>
   <url type="homepage">https://www.anydesk.com</url>
-  <description>
-    <p>Working together? Accessing your apps from the road? Showing family videos to your friends?
-    No problem with AnyDesk. And your data stays at home.</p>
-    <p>AnyDesk is the fastest remote desktop software on the market. It allows for new usage scenarios and applications that have not been possible with current remote desktop software.</p>
-  </description>
+  <description><p>AnyDesk ensures secure and reliable remote desktop connections for IT professionals and on-the-go individuals alike.</p><p>NOTE: This wrapper is not verified by, affiliated with, or supported by Adobe.</p></description>
```
Comment (wjt) [inline com.anydesk.Anydesk.appdata.xml]: "Adobe"?

File: com.anydesk.Anydesk.appdata.xml [hunk 5]
```diff
@@ -1,21 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>com.anydesk.Anydesk.desktop</id>
-  <name>Anydesk</name>
+  <name>AnyDesk</name>
   <project_license>LicenseRef-proprietary</project_license>
-  <developer_name>Anydesk</developer_name>
-  <summary>Your desk at any desk</summary>
+  <developer_name>AnyDesk</developer_name>
```
Comment (wjt) [inline com.anydesk.Anydesk.appdata.xml]:
```suggestion
  <developer_name>AnyDesk Software GmbH</developer_name>
```

perhaps?

File: com.anydesk.Anydesk.appdata.xml [hunk 6]
```diff
@@ -1,21 +1,19 @@
...
-  </description>
+  <description><p>AnyDesk ensures secure and reliable remote desktop connections for IT professionals and on-the-go individuals alike.</p><p>NOTE: This wrapper is not verified by, affiliated with, or supported by Adobe.</p></description>
+  <description xml:lang="pt_BR"><p>A AnyDesk assegura ligações de computador remoto seguras e fiáveis tanto para profissionais de TI como para pessoas em viagem.</p><p>NOTA: Este aplicativo não foi empacotado pelo AnyDesk</p></description>
+  <description xml:lang="es"><p>Gracias a AnyDesk, contará con conexiones seguras y fiables de escritorio remoto para profesionales informáticos y usuarios en movimiento.</p><p>NOTA: Esta aplicación no ha sido empaquetado por AnyDesk</p></description>
   <screenshots>
-    <screenshot>
-      <image type="source">https://anydesk.com/media/images/image-sshot-linux-main-en_scXhgcK.width-990.png</image>
-    </screenshot>
+    <screenshot><image type="source">https://anydesk.com/media/images/image-sshot-linux-main-en_scXhgcK.width-990.png</image></screenshot>
```
Comment (wjt) [inline com.anydesk.Anydesk.appdata.xml]: 404.

**Comment by wjt** [comment]
Superseded by #23.

**Pull request closed without merging**